catalog/justfile

```diff
@@ -142,6 +142,27 @@ generate-dag-docs fail_on_diff="false":
       fi
     fi
 
+
+# Generate the DAG documentation
+generate-media-props fail_on_diff="true":
```

Comment:
I tried moving this workflow into docker, but then realized that we don't mount the local `.sql` files that we parse to generate the db docs into the container. Here's the error I got:
```
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/docker/upstream_db/0006_openledger_audio_schema.sql'
```
Is it possible to open them from the host, @AetherUnbound ? Seems like this would make the setup unnecessarily complex.

Reply:
ohhhh dang, I didn't even think about having to mount those files 😮 We can actually mount those files relatively easily in the recipe at least, it would just require modifying the `just ../run` command here: 

https://github.com/WordPress/openverse/blob/78d799c4d42f2708fbe9149321ff58aec1126d7f/catalog/justfile#L127-L129

Something similar to the way the tests are mounted (note the `--volume` here):

https://github.com/WordPress/openverse/blob/78d799c4d42f2708fbe9149321ff58aec1126d7f/catalog/justfile#L101-L108

Even though it's not needed for the DAG markdown generation, it can be done for all the generation recipes anyway!

Reply:
Thank you, I'll try that.
Not related to this PR, but is the process of keeping those files up-to-date automated?

Reply:
It is not! I believe that's the purview of #416 😄